src/id128/id128.c

```diff
@@ -63,6 +66,112 @@ static int verb_invocation_id(int argc, char **argv, void *userdata) {
         return id128_pretty_print(id, arg_mode);
 }
 
+typedef struct gpt_table_entry {
+        const char *name;
+        const char *cname;
+        sd_id128_t uuid;
+} gpt_table_entry;
+
+static gpt_table_entry gpt_table[] = {
+        { "gpt-root-x86",              "GPT_ROOT_X86",              GPT_ROOT_X86    },
+        { "gpt-root-x86-64",           "GPT_ROOT_X86_64",           GPT_ROOT_X86_64 },
+        { "gpt-root-arm   ",           "GPT_ROOT_ARM",              GPT_ROOT_ARM    },
+        { "gpt-root-arm_64",           "GPT_ROOT_ARM_64",           GPT_ROOT_ARM_64 },
+        { "gpt-root-ia64",             "GPT_ROOT_IA64"  ,           GPT_ROOT_IA64   },
+        { "gpt-esp",                   "GPT_ESP"        ,           GPT_ESP         },
+        { "gpt-xbootldr",              "GPT_XBOOTLDR"   ,           GPT_XBOOTLDR    },
+        { "gpt-swap",                  "GPT_SWAP"       ,           GPT_SWAP        },
+        { "gpt-home",                  "GPT_HOME"       ,           GPT_HOME        },
+        { "gpt-srv",                   "GPT_SRV"        ,           GPT_SRV         },
+        { "gpt-var",                   "GPT_VAR"        ,           GPT_VAR         },
+        { "gpt-tmp",                   "GPT_TMP"        ,           GPT_TMP         },
+
+        { "gpt-root-x86-verity",       "GPT_ROOT_X86_VERITY",       GPT_ROOT_X86_VERITY    },
+        { "gpt-root-x86_64-verity",    "GPT_ROOT_X86_64_VERITY",    GPT_ROOT_X86_64_VERITY },
+        { "gpt-root-arm-verity",       "GPT_ROOT_ARM_VERITY",       GPT_ROOT_ARM_VERITY    },
+        { "gpt-root-arm_64-verity",    "GPT_ROOT_ARM_64_VERITY",    GPT_ROOT_ARM_64_VERITY },
+        { "gpt-root-ia64-verity",      "GPT_ROOT_IA64_VERITY",      GPT_ROOT_IA64_VERITY   },
+
+        { "gpt-root-native",           "GPT_ROOT_NATIVE",           GPT_ROOT_NATIVE },
+#ifdef GPT_ROOT_SECONDARY
+        { "gpt-root-secondary",        "GPT_ROOT_SECONDARY",        GPT_ROOT_SECONDARY },
+#endif
+        { "gpt-root-native-verity",    "GPT_ROOT_NATIVE_VERITY",    GPT_ROOT_NATIVE_VERITY },
+#ifdef GPT_ROOT_SECONDARY_VERITY
+        { "gpt-root-secondary-verity", "GPT_ROOT_SECONDARY_VERITY", GPT_ROOT_SECONDARY_VERITY },
+#endif
+};
```

Comment:
hmm, the `gpt-` prefix you kept here, I presume that's because you want to turn this into a table eventually that lists other uuids, too?

Reply:
Yes, I think we'll want to print other stuff too. I tried to keep everything generic.